library/ssl_misc.h

```diff
@@ -20,6 +20,9 @@
 
 #include "psa/crypto.h"
 #include "psa_util_internal.h"
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3)
```

Comment:
Before it was `#if defined(MBEDTLS_USE_PSA_CRYPTO) || defined(MBEDTLS_SSL_PROTO_TLS1_3)` and `defined(MBEDTLS_USE_PSA_CRYPTO)` is always True thus just no guard.